build/config/SignToolData.json

```diff
@@ -45,7 +46,9 @@
         "Dlls\\RazorServiceHub\\Microsoft.CodeAnalysis.Remote.Razor.ServiceHub.dll",
         "Dlls\\RazorVisualStudio\\Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient.dll",
         "Dlls\\TextEditorFeatures\\Microsoft.CodeAnalysis.EditorFeatures.Text.dll",
+        "Dlls\\TestUtilities\\net461\\Roslyn.Test.Utilities.dll",
```

Comment:
This may be unnecessary now. Originally it was signed because it was a part of the VS.External nupkg. Discussions on slack caused me to no longer sign that nupkg.

Reply:
This is necessary. They are both included in our MS.VS.IntegrationTest.Utilities NuPkg